### PR TITLE
ionio-illustrate:0.1.0

### PR DESCRIPTION
**Pull request by JamesxX**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This package introduces a means by which to display and annotate mass spectrometric data built upon Cetz, which would be useful for documents in chemical or physical disciplines. It is distinct enough that it would not be suitable as a pull request into the Cetz package itself, however, I will be offering my support to the author to implement a combplot chart based on this code.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/mass-spec/0.1.0/manual.typ [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+#set par(justify: true, leading: 0.618em)
+#v(3em)
+
+= Introduction
+This package implements a Cetz chart-like object for displaying mass spectrometric data in Typst documents. It allows for individually styled mass peaks, callouts, titles, and mass calipers.
+
+= Usage
+This is the minimal starting point:
+```typ
+#import "@local/mass-spec:0.1.0": *
```
Comment (istudyatuni) [inline packages/preview/mass-spec/0.1.0/manual.typ:37]:
```suggestion
#import "@preview/mass-spec:0.1.0": *
```

Comment (JamesxX) [reply packages/preview/mass-spec/0.1.0/manual.typ]: Good catch!

**Comment by laurmaedje** [comment]
As is, the name falls a bit too much in the "merely descriptive" category. Could you add a little uniqueness to it in case somebody else also wants to submit a package related to mass spectrometry?

File: packages/preview/mass-spec/0.1.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[package]
+name = "mass-spec"
+version = "0.1.0"
+entrypoint = "mass-spec.typ"
+authors = ["James (Fuzzy) Swift"]
+license = "MIT"
+description = "Mass spectra with annotations for typst."
+exclude = ["/assets/*"]
```
Comment (laurmaedje) [inline packages/preview/mass-spec/0.1.0/typst.toml:8]: The manual should also be excluded here (in particular the PDF) to keep the package size down.

**Review by laurmaedje — COMMENTED** [review]
Currently, the package is lacking a README. This is important because that's the first thing a user will see when they click in the package in the package list. A PDF manual is good for advanced usage details, but a basic overview should be available in the README.

File: packages/preview/mass-spec/0.1.0/LICENSE [hunk 3]
```diff
@@ -0,0 +1,20 @@
+MIT License
+Copyright (c) 2023 Satvik Saha, DVDTSB
```
Comment (laurmaedje) [inline packages/preview/mass-spec/0.1.0/LICENSE:2]: Are the names in the copyright correct or this a copy-paste error?

**Comment by JamesxX** [comment]
> As is, the name falls a bit too much in the "merely descriptive" category. Could you add a little uniqueness to it in case somebody else also wants to submit a package related to mass spectrometry?

I'm afflicted with a deep lack of imagination so I'll run them by you first before changing all the documentation! How does "mass-plotter" sit with you?

File: packages/preview/mass-spec/0.1.0/LICENSE (hunk 3 above)
Comment (JamesxX) [reply packages/preview/mass-spec/0.1.0/LICENSE:2]: Not at all! I'll update :)

File: packages/preview/mass-spec/0.1.0/typst.toml (hunk 2 above)
Comment (JamesxX) [reply packages/preview/mass-spec/0.1.0/typst.toml:8]: Will do

**Comment by JamesxX** [comment]
> Currently, the package is lacking a README. This is important because that's the first thing a user will see when they click in the package in the package list. A PDF manual is good for advanced usage details, but a basic overview should be available in the README.

I'm guessing the format would need to be markdown? I'll get that going.

**Comment by laurmaedje** [comment]
Yes, Markdown is correct. Regarding adding some uniqueness to the name, it helps just to play a little with the words. The list on https://typst.app/docs/packages should give you an idea. mass-plotter would be fine I guess because it leaves more opportunity for other packages in the space, but it's still leaning towards the "merely descriptive" side.

**Comment by JamesxX** [comment]
I won't lie, I've relied heavily on chatgpt to come up with the name

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**